homeassistant/components/reolink/store.py

```diff
@@ -0,0 +1,73 @@
+"""Local storage for the Reolink integration."""
+
+import asyncio
+import logging
+from pathlib import Path
+
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class ReolinkStore:
```

Comment:
Why are you creating your own store object? We have a helper class for this in `helpers/storage.py`.

Reply:
Because local_todo and roborock do it that way and I used those as my examples.....
https://github.com/home-assistant/core/blob/dev/homeassistant/components/roborock/roborock_storage.py
https://github.com/home-assistant/core/blob/dev/homeassistant/components/local_todo/store.py

But if there already is a helper for this that sounds better.

Reply:
Alright, I re-wrote this whole PR to now use the helpers/storage.py
Thanks for this feedback, this is now much cleaner.

Kind of wish I new this earlier.....